lib/mpl_toolkits/axes_grid1/parasite_axes.py

```diff
@@ -258,8 +258,8 @@ def get_aux_axes(self, tr, viewlim_mode="equal", axes_class=None):
         ax2._remove_method = lambda h: self.parasites.remove(h)
         return ax2
 
-
     def _get_legend_handles(self, legend_handler_map=None):
+        # don't use this!
```

Comment:
Is this intended for deprecation (then just delete it, it's private) or is it just used privately (then that's already implied by the underscore and the comment doesn't add anything)?

Reply:
Fair enough - I was cautious because I don't really grok everything in `axes_grid1` and I didn't want just cut some code that someone may use...

Reply:
rebased...

Reply:
@jklymak That is a good instinct!